willyserge/broadcaster
# #169811303  a user should be able to create an account

#### What does this PR do?
     an endpoint where a user should create an account
#### Description of Task to be completed?
    have this endpoint working  /api/v1/auth/signup
   
#### How should this be manually tested?
      after cloning the repo 
- run npm init
- run npm start to start the server
- check your port number
- go to postman 
-  go to http://localhost:YourPortNumber/api/v1/auth/signup

#### What are the relevant pivotal tracker stories?
#169811303
